demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1994,11 +1994,15 @@ def get_all_files_edited_in_pack_ignore(self, modified_files: set) -> set:
                 continue
             # if the repo does not have remotes, get the .pack-ignore content from the master branch in Github
             # if the repo is not in remote / file cannot be found, try to take it from the latest commit on the default branch (usually master/main)
-            old_pack_ignore_content = get_remote_file(
-                old_file_path, "master"
-            ) or self.git_util.get_local_remote_file_content(
-                f"{GitUtil.find_primary_branch(self.git_util.repo)}:{old_file_path}"
-            )
+            old_pack_ignore_content = get_remote_file(old_file_path, "master")
+            if not old_pack_ignore_content:
+                try:
+                    old_pack_ignore_content = self.git_util.get_local_remote_file_content(
+                        f"{GitUtil.find_primary_branch(self.git_util.repo)}:{old_file_path}"
+                    )
+                except GitCommandError:
+                    old_pack_ignore_content = ""
```

Comment:
add a log to the `except` block. Not sure if this is `info` or `warning` though

Reply:
ill do it warning

Reply:
Tell the users how it affects them - is this ok? should it be fixed? What are the implications?

Reply:
done